apps/studio/next.config.mjs

```diff
@@ -33,6 +33,7 @@ const ContentSecurityPolicy = `
     https://vitals.vercel-insights.com/v1/vitals
     https://*.amazonaws.com
     https://placehold.co
+    https://cdn.growthbook.io/api/features/${env.NEXT_PUBLIC_GROWTHBOOK_CLIENT_KEY}
```

Comment:
now we have two env vars that are the same. do you want to remove the (non-public) `GROWTHBOOK_CLIENT_KEY` env var with this public one?

remember update iac also

Reply:
actually i think these are different! not very sure _how_ they're different tbh and growthbook doesn't make it very clear ._.

i was under the impression that you'd need to create 1 for be, 1 for fe and 1 for each env?

<img width="1167" alt="image" src="https://github.com/user-attachments/assets/d23d686d-c023-4f97-8e44-51f0239156cd">

Reply:
don't need actually, but up to you haha. one is enough

Reply:
@karrui I think the FE only is read-only, the BE only is read+write. so should have separate key if we are exposing to FE

Reply:
At least thats what it was in the old version of GB, can double check if the latest release is still the case

Reply:
okie. leave it up to chin!